embc-app/ClientApp/src/app/provincial-admin/components/volunteer-organization-list/volunteer-organization-list.component.ts

```diff
@@ -129,14 +136,17 @@ export class VolunteerOrganizationListComponent implements OnInit, OnDestroy {
       default:
         break;
     }
+
     // go get the collection of meta and data
     // get the volunteers using the parameters supplied
     this.volunteerService.getVolunteers(this.previousQuery)
-      .subscribe((v: ListResult<Volunteer>) => {
+      .subscribe(listResult => {
```

Comment:
It makes sense to favour explicitly added typings instead of using a variable name to track it in typescript. If we make a change to the view models then we lose the hinting that the code is broken at transpile time. Or if we go looking for usage of a particular interface this explicit typing will no longer show up. All of the explicit typings go away at transpile time so for the sake of maintainability and readability leave it in. There is no optimization added by removing typings or comments.

Reply:
This variable is already typed -- just mouse over it and see.
In other cases, yes, it makes sense to add type info.